<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormattingToolbar</class>
 <widget class="QWidget" name="FormattingToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Formatting toolbar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFontComboBox" name="font">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Font list</string>
     </property>
     <property name="statusTip">
      <string>Font list</string>
     </property>
     <property name="whatsThis">
      <string>Font list</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="fontSize">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="increaseSizeFont">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Increase size</string>
       </property>
       <property name="statusTip">
        <string>Increase size</string>
       </property>
       <property name="whatsThis">
        <string>Increase size</string>
       </property>
       <property name="text">
        <string>Increase size</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/IncreaseFontSize</normaloff>:/FormattingToolbar/IncreaseFontSize</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="decreaseSizeFont">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Decrease size</string>
       </property>
       <property name="statusTip">
        <string>Decrease size</string>
       </property>
       <property name="whatsThis">
        <string>Decrease size</string>
       </property>
       <property name="text">
        <string>Decrease size</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/DecreaseFontSize</normaloff>:/FormattingToolbar/DecreaseFontSize</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bold">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="statusTip">
        <string>Bold</string>
       </property>
       <property name="whatsThis">
        <string>Bold</string>
       </property>
       <property name="text">
        <string>Bold</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/bold</normaloff>:/FormattingToolbar/bold</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italic">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Italic</string>
       </property>
       <property name="statusTip">
        <string>Italic</string>
       </property>
       <property name="whatsThis">
        <string>Italic</string>
       </property>
       <property name="text">
        <string>Italic</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/italic</normaloff>:/FormattingToolbar/italic</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorTextButton" name="textColor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Text color</string>
       </property>
       <property name="statusTip">
        <string>Text color</string>
       </property>
       <property name="whatsThis">
        <string>Text color</string>
       </property>
       <property name="text">
        <string>Text color</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/color-text</normaloff>:/FormattingToolbar/color-text</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorBackgroundButton" name="colorBackground">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Color background</string>
       </property>
       <property name="statusTip">
        <string>Color background</string>
       </property>
       <property name="whatsThis">
        <string>Color background</string>
       </property>
       <property name="text">
        <string>Color background</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/color-background</normaloff>:/FormattingToolbar/color-background</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="subScript">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Subscript</string>
       </property>
       <property name="statusTip">
        <string>Subscript</string>
       </property>
       <property name="whatsThis">
        <string>Subscript</string>
       </property>
       <property name="text">
        <string>Subscript</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/subscript</normaloff>:/FormattingToolbar/subscript</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="superScript">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Superscript</string>
       </property>
       <property name="statusTip">
        <string>Superscript</string>
       </property>
       <property name="whatsThis">
        <string>Superscript</string>
       </property>
       <property name="text">
        <string>Superscript</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/super-script</normaloff>:/FormattingToolbar/super-script</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonUnderlineStyleList" name="underline">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Underline</string>
       </property>
       <property name="statusTip">
        <string>Underline</string>
       </property>
       <property name="whatsThis">
        <string>Underline</string>
       </property>
       <property name="text">
        <string>Underline</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/underline</normaloff>:/FormattingToolbar/underline</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="strikeout">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Strikeout</string>
       </property>
       <property name="statusTip">
        <string>Strikeout</string>
       </property>
       <property name="whatsThis">
        <string>Strikeout</string>
       </property>
       <property name="text">
        <string>Strikeout</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/strikeout</normaloff>:/FormattingToolbar/strikeout</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="overline">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Overline</string>
       </property>
       <property name="statusTip">
        <string>Overline</string>
       </property>
       <property name="whatsThis">
        <string>Overline</string>
       </property>
       <property name="text">
        <string>Overline</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/overline</normaloff>:/FormattingToolbar/overline</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lower">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Lower</string>
       </property>
       <property name="statusTip">
        <string>Lower</string>
       </property>
       <property name="whatsThis">
        <string>Lower</string>
       </property>
       <property name="text">
        <string>Lower</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/lower</normaloff>:/FormattingToolbar/lower</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="upper">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Upper</string>
       </property>
       <property name="statusTip">
        <string>Upper</string>
       </property>
       <property name="whatsThis">
        <string>Upper</string>
       </property>
       <property name="text">
        <string>Upper</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/upper</normaloff>:/FormattingToolbar/upper</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="insertHLine">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Insert a horizontal line</string>
       </property>
       <property name="statusTip">
        <string>Insert a horizontal line</string>
       </property>
       <property name="whatsThis">
        <string>Insert a horizontal line</string>
       </property>
       <property name="text">
        <string>Insert a horizontal line</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/hLine</normaloff>:/FormattingToolbar/hLine</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="alignLeft">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Align left</string>
       </property>
       <property name="statusTip">
        <string>Align left</string>
       </property>
       <property name="whatsThis">
        <string>Align left</string>
       </property>
       <property name="text">
        <string>Align left</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/align-left</normaloff>:/FormattingToolbar/align-left</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignCenter">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Align center</string>
       </property>
       <property name="statusTip">
        <string>Align center</string>
       </property>
       <property name="whatsThis">
        <string>Align center</string>
       </property>
       <property name="text">
        <string>Align center</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/align-center</normaloff>:/FormattingToolbar/align-center</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignRight">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Align right</string>
       </property>
       <property name="statusTip">
        <string>Align right</string>
       </property>
       <property name="whatsThis">
        <string>Align right</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Align right</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/align-right</normaloff>:/FormattingToolbar/align-right</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignJustify">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Align justify</string>
       </property>
       <property name="statusTip">
        <string>Align justify</string>
       </property>
       <property name="whatsThis">
        <string>Align justify</string>
       </property>
       <property name="text">
        <string>Align justify</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/align-justify</normaloff>:/FormattingToolbar/align-justify</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="eraser">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Eraser</string>
       </property>
       <property name="statusTip">
        <string>Eraser</string>
       </property>
       <property name="whatsThis">
        <string>Eraser</string>
       </property>
       <property name="text">
        <string>Eraser</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/eraser</normaloff>:/FormattingToolbar/eraser</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonBulletedList" name="bulletedList">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Bulleted list</string>
       </property>
       <property name="statusTip">
        <string>Bulleted list</string>
       </property>
       <property name="whatsThis">
        <string>Bulleted list</string>
       </property>
       <property name="text">
        <string>Bulleted list</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/list-square</normaloff>:/FormattingToolbar/list-square</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonOrderedList" name="orderedList">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Ordered list</string>
       </property>
       <property name="statusTip">
        <string>Ordered list</string>
       </property>
       <property name="whatsThis">
        <string>Ordered list</string>
       </property>
       <property name="text">
        <string>Ordered list</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/list-decimal</normaloff>:/FormattingToolbar/list-decimal</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="insertHyperlink">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Insert hyperlink</string>
       </property>
       <property name="statusTip">
        <string>Insert hyperlink</string>
       </property>
       <property name="whatsThis">
        <string>Insert hyperlink</string>
       </property>
       <property name="text">
        <string>Insert hyperlink</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/hyperlink</normaloff>:/FormattingToolbar/hyperlink</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="insertImage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Insert picture</string>
       </property>
       <property name="statusTip">
        <string>Insert picture</string>
       </property>
       <property name="whatsThis">
        <string>Insert picture</string>
       </property>
       <property name="text">
        <string>Insert picture</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/picture</normaloff>:/FormattingToolbar/picture</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSelectionTableSizes" name="insertTable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Insert table</string>
       </property>
       <property name="statusTip">
        <string>Insert table</string>
       </property>
       <property name="whatsThis">
        <string>Insert table</string>
       </property>
       <property name="text">
        <string>Insert table</string>
       </property>
       <property name="icon">
        <iconset resource="FormattingToolbar.qrc">
         <normaloff>:/FormattingToolbar/table</normaloff>:/FormattingToolbar/table</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="FormattingToolbar.qrc">
     <normaloff>:/FormattingToolbar/bold</normaloff>:/FormattingToolbar/bold</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="FormattingToolbar.qrc">
     <normaloff>:/FormattingToolbar/italic</normaloff>:/FormattingToolbar/italic</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonUnderlineStyleList</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/ButtonUnderlineStyleList.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonBulletedList</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/ButtonBulletedList.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonOrderedList</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/ButtonOrderedList.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonSelectionTableSizes</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/ButtonSelectionTableSizes.h</header>
  </customwidget>
  <customwidget>
   <class>ColorBackgroundButton</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/colorbackgroundbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ColorTextButton</class>
   <extends>QToolButton</extends>
   <header>FormattingToolbar/colortextbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>decreaseSizeFont</tabstop>
 </tabstops>
 <resources>
  <include location="FormattingToolbar.qrc"/>
 </resources>
 <connections/>
</ui>
